src/Compilers/Test/Utilities/VisualBasic/TestOptions.vb

```diff
@@ -7,7 +7,10 @@ Imports Microsoft.CodeAnalysis.Diagnostics
 Imports Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 Imports Roslyn.Test.Utilities
 
-Public Class TestOptions
+Public NotInheritable Class TestOptions
+    Private Sub New()
+    End Sub
```

Comment:
❓ What's the difference between this approach and changing `Class` &rarr; `Module`?

Reply:
@sharwell I preferred class since module members are globally accessible when the module is in scope.